CFLAGS and LDFLAGS aren't properly formatted for the makefile, resulting in "invalid option" errors.

--- GNUmakefile.original	2022-09-01 10:31:46.000000000 +0100
+++ GNUmakefile	2022-09-01 10:45:14.000000000 +0100
@@ -101,7 +101,7 @@
 else
 ifdef LEGACY
 	$(MAKE) -C $(TMPLEGACY)
-	$(MAKE) -C $(TMPLEGACY) PREFIX=$(realpath $(LEGACYLIBS)) CFLAGS=$(CFLAGS) LDFLAGS=$(LDFLAGS) install
+	$(MAKE) -C $(TMPLEGACY) PREFIX=$(realpath $(LEGACYLIBS)) install
 	rm -rf $(TMPLEGACY)
 	$(eval override LDFLAGS+=-L$(realpath $(LEGACYLIBS))/lib -lMacportsLegacySupport)
 endif
